pywinauto/application.py

```diff
@@ -901,7 +901,7 @@ def connect(self, **kwargs):
             connected = True
 
         elif 'path' in kwargs:
-            self.process = process_from_module(kwargs['path'])
+            self.process = process_from_module(kwargs['path'], timeout=timeout)

```

Comment:
It's better to call `timings.wait_until_passes` or `timings.wait_until` to avoid new param introduction.